src/kirby/components/divider/divider.component.scss

```
@@ -1,10 +1,18 @@
 @import '../../scss/utils';
 
+:host-context(kirby-card) {
```

Consider merging the 2 selectors:
```suggestion
:host-context(kirby-card),
:host-context(.kirby-color-brightness-white) {
  --kirby-divider-color: #{get-color('background-color')};
}

```

---

Done